helm/helm.go

```diff
@@ -233,12 +234,16 @@ func templateChart(ctx *ankh.ExecutionContext, chart ankh.Chart, ankhFile ankh.A
 	if ctx.Mode == ankh.Explain {
 		return explain(helmCmd.Args), nil
 	}
+	var stdout, stderr bytes.Buffer
+	helmCmd.Stdout = &stdout
+	helmCmd.Stderr = &stderr
 
-	helmOutput, err := helmCmd.CombinedOutput()
```

Comment:
was helm previously writing to stderr without returning an `err`? sadness

Reply:
yerp, non fatal errors

Reply:
Good catch